<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=generator content="Hugo 0.103.0-DEV"><meta name=author content="Andrey Akinshin"><link href=/img/favicon.ico rel=icon type=image/x-icon><meta name=keywords content><title>Research projects | Andrey Akinshin</title><meta name=description content="Research projects Google Scholar ORCID ResearchGate arXiv Web of Science Scopus This page aggregates results from some of my research projects:
Statistical..."><meta name=twitter:site content="@andrey_akinshin"><meta name=twitter:creator content="@andrey_akinshin"><script>localStorage.getItem("color-theme")==="dark"||!("color-theme"in localStorage)&&window.matchMedia("(prefers-color-scheme: dark)").matches?document.documentElement.classList.add("dark"):document.documentElement.classList.remove("dark")</script><link href=https://aakinshin.net/css/main.min.e75ae85f116f2518a5f5161199a56c6e1ec12c760cdaf78f0d207d929fce8839.css rel=stylesheet type=text/css media=all><link rel=alternate type=application/rss+xml href=https://aakinshin.net/posts/index.xml title="RSS Feed"></head><body><nav><div class="container flex flex-wrap items-center justify-between mx-auto max-w-6xl h-12 px-6"><div class="flex items-center justify-start justify-self-start"><div class="flex nav-item h-12 w-12 items-center justify-center"><a class=text-white href=https://aakinshin.net/><svg class="fai w-5 h-5 pr-0 mb-0.5"><use xlink:href="/img/fa/all.svg#house-chimney"/></svg></a></div><div class="flex nav-item h-12 items-center"><a class="px-3 text-white" href=https://aakinshin.net/posts/>Posts</a></div><div class="flex nav-item h-12 items-center"><a class="px-3 text-white" href=https://aakinshin.net/research/>Research</a></div><div class="flex nav-item h-12 items-center"><a class="px-3 text-white" href=https://aakinshin.net/about/>About</a></div></div><button id=theme-toggle type=button class="nav-item p-3 content-center justify-self-end"><svg id="theme-toggle-dark-icon" class="hidden w-6 h-6" fill="currentcolor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001.0 1010.586 10.586z"/></svg><svg id="theme-toggle-light-icon" class="hidden w-6 h-6" fill="currentcolor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M10 2a1 1 0 011 1v1A1 1 0 119 4V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707A1 1 0 1113.536 5.05l.707-.707a1 1 0 011.414.0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707A1 1 0 004.343 5.757l.707.707zm1.414 8.486-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z" fill-rule="evenodd" clip-rule="evenodd"/></svg></button></div></nav><div class="main container mx-auto max-w-6xl px-6"><div class=main-content><h1 id=research-projects>Research projects</h1><div class="flex flex-wrap gap-y-1 mb-5"><a class=label-link href="https://scholar.google.com/citations?hl=en&user=rYVl83IAAAAJ&view_op=list_works&sortby=pubdate"><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#graduation-cap"/></svg>Google Scholar</a>
<a class=label-link href=https://orcid.org/0000-0003-3553-9367><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#orcid"/></svg>ORCID</a>
<a class=label-link href=https://www.researchgate.net/profile/Andrey_Akinshin><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#researchgate"/></svg>ResearchGate</a>
<a class=label-link href=https://arxiv.org/a/akinshin_a_1.html><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a class=label-link href=https://www.webofscience.com/wos/author/record/2102893><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</a>
<a class=label-link href="https://www.scopus.com/authid/detail.uri?authorId=56826126900"><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</a></div><p>This page aggregates results from some of my research projects:</p><ul><li><a href=#statistics>Statistical performance analysis</a></li><li><a href=#benchmarking>Software benchmarking</a></li><li><a href=#gene-networks>Mathematical models of gene networks</a></li><li><a href=#signal-processing>Digital signal processing</a></li></ul><hr><h2 id=statistics>Statistical performance analysis</h2><p>This project aims to develop advanced statistical procedures
for automatic analysis of software performance measurements.
This included various
location estimators,
dispersion estimators,
quantile estimators,
effect size estimators,
density estimators,
change point detectors,
outlier detectors,
and multimodality detectors.
The goal is to provide a set of non-parametric, robust, and efficient approaches
that are applicable to software performance distributions.</p><p>Every Tuesday, I share my preliminary research notes in the form of
<a href=https://aakinshin.net/tags/statistics/>blog posts (147)</a>.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin</i> <span title>“Weighted quantile estimators”</span> (2023) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/arXiv.2304.07265>10.48550/arXiv.2304.07265</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/arXiv.2304.07265 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2304.07265 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/2304.07265.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-wqe class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg>GitHub</a></div></li><li><i>Andrey Akinshin</i> <span title>“Finite-sample Rousseeuw-Croux scale estimators”</span> (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/arXiv.2209.12268>10.48550/arXiv.2209.12268</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/arXiv.2209.12268 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2209.12268 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/2209.12268.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-frc class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg>GitHub</a></div></li><li><i>Andrey Akinshin</i> <span title>“Quantile absolute deviation”</span> (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/ARXIV.2208.13459>10.48550/ARXIV.2208.13459</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/ARXIV.2208.13459 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2208.13459 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/2208.13459.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-qad class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg>GitHub</a></div></li><li><i>Andrey Akinshin</i> <span title>“Finite-sample bias-correction factors for the median absolute deviation based on the Harrell-Davis quantile estimator and its trimmed modification”</span> (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/ARXIV.2207.12005>10.48550/ARXIV.2207.12005</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/ARXIV.2207.12005 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2207.12005 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/2207.12005.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-mad-factors class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg>GitHub</a></div></li><li><i>Andrey Akinshin</i> <span title='Traditional quantile estimators that are based on one or two order statistics are a common way to estimate distribution quantiles based on the given samples. These estimators are robust, but their statistical efficiency is not always good enough. A more efficient alternative is the Harrell-Davis quantile estimator which uses a weighted sum of all order statistics. Whereas this approach provides more accurate estimations for the light-tailed distributions, it’s not robust. To be able to customize the tradeoff between statistical efficiency and robustness, we could consider a trimmed modification of the Harrell-Davis quantile estimator. In this approach, we discard order statistics with low weights according to the highest density interval of the beta distribution.'>“Trimmed Harrell-Davis quantile estimator based on the highest density interval of the given width”</span> (2022) // Communications in Statistics - Simulation and Computation. Publisher: Taylor & Francis. Pp.&nbsp;1-11. DOI:&nbsp;<a href=https://doi.org/10.1080/03610918.2022.2050396>10.1080/03610918.2022.2050396</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1080/03610918.2022.2050396 class="label-link text-sm">DOI</a>
<a href=https://www.tandfonline.com/doi/abs/10.1080/03610918.2022.2050396 class="label-link text-sm">Link</a>
<a href=https://arxiv.org/abs/2111.11776 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/2111.11776.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-thdqe class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg>GitHub</a></div></li></ol><hr><h2 id=benchmarking>Software benchmarking</h2><p>This project aims to provide a set of reliable tools and approaches for software benchmarking.</p><p>Since 2013, I have been working on developing
<a href=https://github.com/dotnet/BenchmarkDotNet>BenchmarkDotNet</a>
(8K+ <a href=https://github.com/dotnet/BenchmarkDotNet/stargazers>GitHub stars</a>,
15K+ <a href="https://github.com/dotnet/BenchmarkDotNet/network/dependents?package_id=UGFja2FnZS0xNTY3MzExMzE%3D">dependent GitHub projects</a>,
18M+ <a href=https://www.nuget.org/packages/BenchmarkDotNet/>NuGet downloads</a>).
This .NET library helps to transform methods into benchmarks,
track their performance,
and share reproducible measurement experiments.
In the context of the <a href=#statistics>statistical performance analysis project</a>,
I&rsquo;m working on the next-generation statistical engine for performance measurement analysis.</p><p>I also wrote a book <a href=https://aakinshin.net/prodotnetbenchmarking/>Pro .NET Benchmarking</a>
about good practices of performance measurements.
This book mostly covers
general benchmarking practices,
performance measurement analysis approaches,
and environmental factors that can affect obtained measurements.
It also contains dozens of case studies that demonstrate common benchmarking pitfalls and explain how to avoid them.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin</i> <span title>“Pro .NET Benchmarking”</span> (2019) // Publisher: Apress. ISBN:&nbsp;978-1-4842-4940-6. Pp.&nbsp;662. DOI:&nbsp;<a href=https://doi.org/10.1007/978-1-4842-4941-3>10.1007/978-1-4842-4941-3</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Book</span>
<span class="label text-sm">Springer</span>
<a href=https://dx.doi.org/10.1007/978-1-4842-4941-3 class="label-link text-sm">DOI</a>
<a href=https://link.springer.com/book/10.1007/978-1-4842-4941-3 class="label-link text-sm">Springer</a>
<a href=https://www.apress.com/us/book/9781484249406 class="label-link text-sm">Apress</a></div></li><li><i>Andrey Akinshin</i> <span title>“The hard world of microbenchmarking”</span> (2017) // Logeek Magazine. Vol.&nbsp;3. Pp.&nbsp;10–15.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<a href=https://career.luxoft.com/upload/iblock/74f/2283_b2e_logeek_magazine_3_a4_2017_02_27_net.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li></ol><hr><h2 id=gene-networks>Mathematical models of gene networks</h2><p>This project aims to describe the mathematical model of various gene networks:
build systems of differential equations that simulate biological systems;
analyze corresponding dynamical systems;
find stationary points, limit cycles, and bifurcations;
develop software for numerical simulations.
The latest work is dedicated to the central regulatory circuit of the morphogenesis system of D. Melanogaster.</p><p>I was working on this project with fellows from
<a href=https://en.wikipedia.org/wiki/Sobolev_Institute_of_Mathematics>Sobolev Institute of Mathematics SB RAS</a> and
<a href=https://www.icgbio.ru/en/>Institute of Cytology and Genetics SB RAS</a>.
It was also the primary research topic of my Ph.D. thesis
&ldquo;Mathematical and numerical modeling of gene network artificial regulatory circuits.&rdquo;</p><p><strong>Selected publications:</strong></p><ol reversed><li><i>V. P. Golubyatnikov, A. A. Akinshin, N. B. Ayupova, L. S. Minushkina</i> <span title>“Stratifications and foliations in phase portraits of gene network models”</span> (2023) // Vavilov Journal of Genetics and Breeding. Publisher: Institute of Cytology and Genetics, SB RAS. Vol.&nbsp;26. No&nbsp;8. Pp.&nbsp;758–764. DOI:&nbsp;<a href=https://doi.org/10.18699/vjgb-22-91>10.18699/vjgb-22-91</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<a href=https://dx.doi.org/10.18699/vjgb-22-91 class="label-link text-sm">DOI</a></div></li><li><i>A. A. Akinshin, N. B. Ayupova, V. P. Golubyatnikov, N. E. Kirillova, O. A. Podkolodnaya, N. L. Podkolodnyy</i> <span title='For a model of a circadian oscillator presented in the form of a 6-dimensional nonlinear dynamical system, conditions of uniqueness of an equilibrium point and conditions of existence of a periodic trajectory (cycle) are established. A client-server application is developed to perform numerical experiments with this model on a cloud server and to visualize the results of these experiments.'>“On a Numerical Model of a Circadian Oscillator”</span> (2022) // Numerical Analysis and Applications. Vol.&nbsp;15. No&nbsp;3. Pp.&nbsp;187-196. DOI:&nbsp;<a href=https://doi.org/10.1134/S1995423922030016>10.1134/S1995423922030016</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1134/S1995423922030016 class="label-link text-sm">DOI</a>
<a href=https://rdcu.be/cUri9 class="label-link text-sm">Link</a></div></li><li><i>V. Golubyatnikov, A. Akinshin, N. Ayupova, N. Kirillova, N. Podkolodnyy, O. Podkolodnaya</i> <span title>“Mathematical and numerical modelling of the circadian oscillator”</span> (2022) // Bioinformatics of Genome Regulation and Structure/Systems Biology (BGRS/SB-2022). Pp.&nbsp;1096–1096. DOI:&nbsp;<a href=https://doi.org/10.18699/SBB-2022-657>10.18699/SBB-2022-657</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.18699/SBB-2022-657 class="label-link text-sm">DOI</a></div></li><li><i>T.A. Bukharina, A.A. Akinshin, V.P. Golubyatnikov, D.P. Furman</i> <span title>“A Model of one central regulatory circuit”</span> (2020) // The Twelfth International Multiconference 'Bioinformatics of genome regulation and structure/systems biology (BGRS/SB-2020)'. Publisher: Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of Sciences, Novosibirsk State University. Pp.&nbsp;143–144. DOI:&nbsp;<a href=https://doi.org/10.18699/BGRS/SB-2020-090>10.18699/BGRS/SB-2020-090</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.18699/BGRS/SB-2020-090 class="label-link text-sm">DOI</a>
<a href=https://publ.icgbio.ru/wp-content/uploads/2020/07/BGRS2020_15.07-144-145.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>T. A. Bukharina, A. A. Akinshin, V. P. Golubyatnikov, D. P. Furman</i> <span title='The results are presented of mathematical and computer simulation of the functioning of the central regulatory circuit (CRC) which is the system integrator of the gene networks ofmorphogenesis of Drosophila mechanoreceptors. The main element of the CRC is represented bythe complex of Achaete-Scute (AS-C) genes, the main genes of the mechanoreceptor morphogenesis.The content level of the proteins encoded by the AS-Cgenes is a determining factor for initiating the development of a mechanoreceptor. We carried outa comparative study of the CRC behavior under normal conditions and in the presence ofmutational changes in the Achaete-Scute complex. Theresults of simulation are in good accord with the available biological data.'>“Mathematical and Numerical Models of the Central Regulatory Circuit of the Morphogenesis System of Drosophila”</span> (2020) // Journal of Applied and Industrial Mathematics. Vol.&nbsp;14. No&nbsp;2. Pp.&nbsp;249-255. DOI:&nbsp;<a href=https://doi.org/10.1134/S1990478920020040>10.1134/S1990478920020040</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1134/S1990478920020040 class="label-link text-sm">DOI</a>
<a href=https://rdcu.be/b5yyl class="label-link text-sm">Link</a>
<a href=https://link.springer.com/article/10.1134/S1990478920020040 class="label-link text-sm">Springer</a></div></li><li><i>A.A. Akinshin, V. P. Golubyatnikov, M.V. Kazantsev</i> <span title>“On discrete structures in phase portraits of some non-linear dynamical systems”</span> (2016) // Siberian Electronic Mathematical Reports. Vol.&nbsp;13. Pp.&nbsp;A.23–A.24. DOI:&nbsp;<a href=https://doi.org/10.17377/semi.2016.13.020>10.17377/semi.2016.13.020</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.17377/semi.2016.13.020 class="label-link text-sm">DOI</a>
<a href=http://semr.math.nsc.ru/v13/a1-41.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>A.A. Akinshin, M.V. Kazantsev</i> <span title='Existence and location of periodical trajectories are important properties of multi-dimensional dynamical systems. In particular, this relates to systems that model gene networks. We can reduce the areas where cyclic trajectories may be located due to analysis of some discrete structures describing the dynamical system, such as state transition diagrams'>“State transition diagrams in hypothetical models of cyclic gene networks functioning”</span> (2015) // 5th International Young Scientists School “System Biology and Bioinformatics”. Novosibirsk. Pp.&nbsp;1.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/SBB2015/en class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> <span title>“A model of one biological 2-cells complex”</span> (2014) // Geometry days in Novosibirsk - 2014: Abstracts of the International Conference. Novosibirsk. ISBN:&nbsp;978-5-86134-145-5. Pp.&nbsp;85–86.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> <span title>“An Extended Model of D. Melanogaster Machrochaete Morphogenesis Gene Network”</span> (2014) // International Conference «Mathematical Modeling and High Performance Computing in Bioinformatics, Biomedicine and Biotechnology». Pp.&nbsp;10.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/mmhpc2014/ class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> <span title>“Modeling of Two-Cells Complex in Morphogenesis of D. Melanogaster Mechanoreceptors”</span> (2014) // International Conference «Mathematical Modeling and High Performance Computing in Bioinformatics, Biomedicine and Biotechnology». Pp.&nbsp;11.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, V. A. Likhoshvai</i> <span title='Gene networks are sets of genes functioning in a coordinated manner. Their major components are the biopolymers DNA, RNA, and proteins. In this paper, we consider a small network with several genes (they can be considered standalone) and observe the concentration of the corresponding proteins. Researching the dynamic behavior of gene networks is a complicated problem. Therefore, we research the principles of their functioning on special hypothetical model objects. These models can be described by a nonlinear differential system. Symmetry is a very important property of living systems. The phenomenon of symmetry can be observed in completely different biological models. The analysis of gene networks model symmetry allows us to construct special methods to solve such standard problems as the problem of existence and stability of attractors, to build the graphical representation of the gene network phase portraits, to determine equilibrium points, to simulate trajectory, etc. Also we performed computer modeling and conducted simulation experiments with our models in R language to verify obtained results.'>“Symmetry in gene network models”</span> (2014) // The 12th Bioinformatics Research and Education Workshop. Bielefeld, Germany. Center for Biotechnology — CeBiTec. Pp.&nbsp;1–5.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://wiki.techfak.uni-bielefeld.de/didy/brew class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> <span title='We consider the phase portrait of a 6-dimensional nonlinear dynamical system considered as a model of an early stage of morphogenesis of D. melanogaster, i.e., appearance of its parental cell in the proneural cluster. Sufficient conditions of existence of two stable equilibrium points of this system are described, and biological interpretation is given.'>“Mathematical modeling of interaction of two cells in proneural cluster of D.mealnogaster (In Russian)”</span> (2014) // Vestnik NSU [NSU Herald]. Novosibirsk. Vol.&nbsp;14. No&nbsp;4. Pp.&nbsp;3-10.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper&jrnid=vngu&paperid=350&option_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> <span title>“Oscillating trajectories in some nonlinear dynamical systems”</span> (2013) // International Conference «Differential Equations. Function Spaces. Approximation Theory». Novosibirsk. IM SB RAS. Pp.&nbsp;311.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> <span title>“Analysis of phase portraits in some gene networks models”</span> (2013) // 5th International Young Scientists School “System Biology and Bioinformatics”. Pp.&nbsp;27.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> <span title='Gene networks are sets of genes functioning in a coordinated manner, their major components are the biopolymers DNA, RNA, and proteins. In this work, we consider a small network with several genes (they can be considered standalone) and observe the concentration of the corresponding proteins. Studying the dynamic behavior of gene networks is a complicated problem. Therefore, we research the principles of their functioning on special hypothetical model. These models can be describe by nonlinear system of differential equations. We are very interested in the questions on existence and stability of attractors in such models. Particular attention is paid to some multidimensional symmetric systems with negative and positive feedbacks. These systems are studied with a specially developed algorithm, which reduces them to a single equation with delayed argument. Cycles in this equation searched by Andronov-Hopf theorem and stability analyzes using the first Lyapunov coefficient. Computer modelling and numerical experiments with our models has been implemented in our special computer program called PhasePortraitAnalyzer. The mathematical core is developed in R, and the interface part is created using C{\#} + WPF. Its software can build the graphical representation of the gene networks phase portraits, search equilibrium points, simulate the trajectory and perform other calculations for analyzing the target system. We can prove the correctness of the numerical solutions using some theorems on topology of the models.'>“Numerical analysis of gene networks models”</span> (2013) // 8th FEBS Congress, Saint Petersburg, Russia, July 6-11, 2013. Vol.&nbsp;280. Pp.&nbsp;547. DOI:&nbsp;<a href=https://doi.org/10.1111/febs.12396>10.1111/febs.12396</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<a href=https://dx.doi.org/10.1111/febs.12396 class="label-link text-sm">DOI</a>
<a href="https://apps.webofknowledge.com/full_record.do?product=UA&search_mode=GeneralSearch&qid=1&SID=U1kYDgZoTuCyURIaqPx&page=1&doc=4" class="label-link text-sm">Web of Science</a>
<a href=https://publons.com/publon/34688524/ class="label-link text-sm">Publons</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, V. A. Likhoshvai</i> <span title>“Mathematical and computational models of gene networks functioning”</span> (2013) // The 11th Bioinformatics Research and Education Workshop. Berlin, Germany. Computational Molecular Biology Department, Max Planck Institute for Molecular Genetics and IMPRS-CBSC.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://cmb.molgen.mpg.de/brew2013/ class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin</i> <span title='We study the occurrence of Andronov-Hopf bifurcation cycles in a neighborhood of stationary points of nonlinear delay equations: we formulate conditions for the existence of a bifurcation, find the bifurcation values, and analyze the stability of the bifurcation cycles.'>“Andronov-Hopf bifurcation for some nonlinear delay equations (In Russian)”</span> (2013) // Sibirskii Zhurnal Industrial'noi Matematiki [Siberian Journal of Industrial Mathematics]. Novosibirsk. Vol.&nbsp;XVI. No&nbsp;55. Pp.&nbsp;3–15.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://elibrary.ru/item.asp?id=20379316" class="label-link text-sm">RSCI</a>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper&jrnid=sjim&paperid=788&option_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, I. V. Golubyatnikov</i> <span title>“On some multidimensional models of gene network functioning”</span> (2013) // Journal of Applied and Industrial Mathematics. Publisher: Springer US. Vol.&nbsp;7. No&nbsp;3. Pp.&nbsp;296–301. DOI:&nbsp;<a href=https://doi.org/10.1134/S1990478913030022>10.1134/S1990478913030022</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1134/S1990478913030022 class="label-link text-sm">DOI</a>
<a href="http://elibrary.ru/item.asp?id=20455452" class="label-link text-sm">RSCI</a>
<a href=http://link.springer.com/article/10.1134/S1990478913030022 class="label-link text-sm">Springer</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> <span title>“Non-uniqueness of cycles in gene networks models”</span> (2012) // The eighth international conference on bioinformatics of genome regulation and structure / systems biology. ISBN:&nbsp;978-5-91291-011. Pp.&nbsp;28.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://www.bionet.nsc.ru/meeting/bgrs2012/BGRS2012_Proceedings.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, Yu. A. Gaidov, I. V. Golubyatnikov</i> <span title>“Unstable cycles in gene networks models”</span> (2012) // The eighth international conference on bioinformatics of genome regulation and structure / systems biology. ISBN:&nbsp;978-5-91291-011. Pp.&nbsp;29.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://www.bionet.nsc.ru/meeting/bgrs2012/BGRS2012_Proceedings.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> <span title>“On Nonuniqueness of Cycles in Dissipative Dynamical Systems of Chemical Kinetics”</span> (2012) // VI-th international conference Solitons, Collapses and Turbulence: Achievements, Developments and Perspectives.. Pp.&nbsp;71–72.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/files/conferences/sct2012/abstracts/114645/128115/Golubyatnikov.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>A. A. Akinshin</i> <span title>“Computer analysis of phase portraits in gene networks models”</span> (2012) // Abstracts of Young scientist's school “Bioinformatics ans systems biology”. Pp.&nbsp;13.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> <span title>“Research of discrete structures in some cyclic dynamical systems (In Russian)”</span> (2012) // Polzunovskiy Vestnik [Polzunov Herald]. Barnaul. Vol.&nbsp;4. Pp.&nbsp;214–218.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href=http://elib.altstu.ru/elib/books/Files/pv2012_04/pdf/214akinshin.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a>
<a href="http://elibrary.ru/item.asp?id=18947027" class="label-link text-sm">RSCI</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> <span title='We show non-uniqueness of cycles in phase portraits of some odd-dimensional nonlinear dynamical systems considered as models of gene networks regulated by negative feedbacks. We find geometric and analytic characteristics of these cycles and construct a graph, which describes qualitative behavior of trajectories of these dynamical systems.'>“Geometric characteristics of cycles in some symmetric dynamical systems (In Russian)”</span> (2012) // Vestnik NSU [NSU Herald]. Series “Mathematics, mechanics, and informatics'”. Novosibirsk. Vol.&nbsp;12. No&nbsp;2. Pp.&nbsp;3–12.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://elibrary.ru/item.asp?id=18756520" class="label-link text-sm">RSCI</a>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper&jrnid=vngu&paperid=114&option_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, I. V. Golubyatnikov</i> <span title='We find conditions of existence of stable cycles in some models of gene networks regulated by negative feedbacks and by simple combinations of negative and positive feedbacks. Special algorithms and programs for numerical simulations of these results are elaborated as well.'>“Mathematical and numerical modeling of gene network functioning”</span> (2011) // International Conference “Modern Problems of Mathematics, Informatics and Bioinformatics”, devoted to the 100th anniversary of professor Alexei A. Lyapunov. Novosibirsk. ISBN:&nbsp;978-5-905569-03-6. Pp.&nbsp;81.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/Lyap-100/ru/reportview/48237 class="label-link text-sm">Conference site</a>
<a href=http://conf.nsc.ru/files/conferences/Lyap-100/89299/Program-Thezises.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li></ol><hr><h2 id=signal-processing>Digital signal processing</h2><p>I was working on this project during my
postdoctoral research fellowship at the
<a href=https://www.weizmann.ac.il/>Weizmann Institute of Science</a>.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin, Gil Goldman, Yosef Yomdin</i> <span title='We consider a reconstruction problem for “spike-train” sig nals F of the a priori known form F ( x ) = ∑ d j =1 a j $\delta$ ( x − x j ) , from their moments m k ( F ) = ∫ x k F ( x ) dx. We assume m k ( F ) to be known for k = 0 , 1 , . . . , 2 d − 1 , with an absolute error not exceeding ǫ {\textgreater} 0. This problem is essentially equivalent to solving the Prony system ∑ d j =1 a j x k j = m k ( F ) , k = 0 , 1 , . . . , 2 d − 1 . We study the “geometry of error amplification” in reconstruc tion of F from m k ( F ) , in situations where the nodes x 1 , . . . , x d near-collide, i.e. form a cluster of size h ≪ 1. We show that in this case error amplification is governed by the “Prony leaves” S q F, in the parameter space of signals F , along which the first q + 1 moments remain constant. On this base we produce accurate , up to the constants, lower and upper bounds on the worst case r econstruction error, and show how the Prony leaves can help in improving rec onstruction accuracy.'>“Geometry of error amplification in solving the Prony system with near-colliding nodes”</span> (2021) // Mathematics of Computation. Vol.&nbsp;90. No&nbsp;327. Pp.&nbsp;267–302. DOI:&nbsp;<a href=https://doi.org/10.1090/mcom/3571>10.1090/mcom/3571</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1090/mcom/3571 class="label-link text-sm">DOI</a>
<a href=https://publons.com/publon/39312469/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1701.04058 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/1701.04058.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>Andrey Akinshin, Gil Goldman, Vladimir Golubyatnikov, Yosef Yomdin</i> <span title='We consider a signal reconstruction problem for signals {\$}F{\$} of the form {\$} F(x)=\backslashsum_{\{}j=1{\}}{\^{}}{\{}d{\}}a_{\{}j{\}}\backslashdelta\backslashleft(x-x_{\{}j{\}}\backslashright),{\$} from their moments {\$}m_k(F)=\backslashint x{\^{}}kF(x)dx.{\$} We assume {\$}m_k(F){\$} to be known for {\$}k=0,1,\backslashldots,N,{\$} with an absolute error not exceeding {\$}\backslashepsilon {\textgreater} 0{\$}. We study the ' geometry of error amplification' in reconstruction of {&#92;$}f{&#92;$} from {&#92;$}m_k(f),{&#92;$} in situations where two neighboring nodes {&#92;$}x_i{&#92;$} and {&#92;$}x_{&#92;{}i+1{&#92;}}{&#92;$} near-collide, i.e {&#92;$}x_{&#92;{}i+1{&#92;}}-x_i=h &#92;backslashll 1{&#92;$}. we show that the error amplification is governed by certain algebraic curves {&#92;$}s_{&#92;{}f,i{&#92;}},{&#92;$} in the parameter space of signals {&#92;$}f{&#92;$}, along which the first three moments {&#92;$}m_0,m_1,m_2{&#92;$} remain constant.'>“Accuracy of reconstruction of spike-trains with two near-colliding nodes”</span> (2017) // 7th International Conference on Complex Analysis and Dynamical Systems (CA&DS VII). Vol.&nbsp;699. Pp.&nbsp;1–17. DOI:&nbsp;<a href=https://doi.org/10.1090/conm/699/14078>10.1090/conm/699/14078</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1090/conm/699/14078 class="label-link text-sm">DOI</a>
<a href=https://publons.com/publon/34688522/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1701.01482 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/1701.01482.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li><li><i>A. Akinshin, D. Batenkov, Y. Yomdin</i> <span title='We consider Fourier reconstruction problem for signals F, which are linear combinations of shifted delta-functions. We assume the Fourier transform of F to be known on the frequency interval [-N,N], with an absolute error not exceeding e {\textgreater} 0. We give an absolute lower bound (which is valid with any reconstruction method) for the ' worst case' reconstruction error of f in situations where the nodes (i.e. the positions of the shifted delta-functions in f) are known to form an l elements cluster of a size h {&#92;textless}{&#92;textless} 1. using 'decimation' reconstruction algorithm we provide an upper bound for the reconstruction error, essentially of the same form as the lower roughly, our main result states that for n*h of order of (2l-1)-st root of e the worst case reconstruction error of the cluster nodes is of the same order as h, and hence the inside configuration of the cluster nodes (in the worst case scenario) cannot be reconstructed at all. on the other hand, decimation algorithm reconstructs f with the accuracy of order of 2l-st root of e.'>“Accuracy of spike-train Fourier reconstruction for colliding nodes”</span> (2015) // Sampling Theory and Applications (SampTA), 2015 International Conference on. Publisher: IEEE. Washington, DC. ISBN:&nbsp;978-1-4673-7353-1. No&nbsp;264. Pp.&nbsp;617–621. DOI:&nbsp;<a href=https://doi.org/10.1109/SAMPTA.2015.7148965>10.1109/SAMPTA.2015.7148965</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg>Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg>Scopus</span>
<a href=https://dx.doi.org/10.1109/SAMPTA.2015.7148965 class="label-link text-sm">DOI</a>
<a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=7148965" class="label-link text-sm">IEEE</a>
<a href=https://publons.com/publon/14766954/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1502.06932 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>arXiv</a>
<a href=https://arxiv.org/pdf/1502.06932.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg>PDF</a></div></li></ol></div></div><script>var themeToggleDarkIcon=document.getElementById("theme-toggle-dark-icon"),themeToggleBtn,themeToggleLightIcon=document.getElementById("theme-toggle-light-icon");const imagesDark=document.querySelectorAll(".img-dark"),imagesLight=document.querySelectorAll(".img-light");localStorage.getItem("color-theme")==="dark"||!("color-theme"in localStorage)&&window.matchMedia("(prefers-color-scheme: dark)").matches?(themeToggleLightIcon.classList.remove("hidden"),imagesDark.forEach(e=>{e.classList.remove("hidden")})):(themeToggleDarkIcon.classList.remove("hidden"),imagesLight.forEach(e=>{e.classList.remove("hidden")})),themeToggleBtn=document.getElementById("theme-toggle"),themeToggleBtn.addEventListener("click",function(){themeToggleDarkIcon.classList.toggle("hidden"),themeToggleLightIcon.classList.toggle("hidden"),imagesLight.forEach(e=>{e.classList.toggle("hidden")}),imagesDark.forEach(e=>{e.classList.toggle("hidden")}),localStorage.getItem("color-theme")?localStorage.getItem("color-theme")==="light"?(document.documentElement.classList.add("dark"),localStorage.setItem("color-theme","dark")):(document.documentElement.classList.remove("dark"),localStorage.setItem("color-theme","light")):document.documentElement.classList.contains("dark")?(document.documentElement.classList.remove("dark"),localStorage.setItem("color-theme","light")):(document.documentElement.classList.add("dark"),localStorage.setItem("color-theme","dark"))})</script><footer class="z-20 p-4 md:p-6 mt-3 w-full border-t border-gray-200 shadow flex items-center justify-center dark:border-gray-600 bg-white dark:bg-zinc-800"><span class="dark:text-gray-400 text-center">© 2013—2023 <span class=whitespace-nowrap>Andrey Akinshin</span></span><ul class="flex flex-wrap items-center px-4 mb-1"><li><a href=https://github.com/AndreyAkinshin><svg class="fai fai-link w-5 h-5"><title>GitHub</title><use xlink:href="/img/fa/all.svg#github"/></svg></a></li><li><a href=https://twitter.com/andrey_akinshin><svg class="fai fai-link w-5 h-5"><title>Twitter</title><use xlink:href="/img/fa/all.svg#twitter"/></svg></a></li><li><a href=https://aakinshin.net/posts/index.xml><svg class="fai fai-link w-5 h-5"><title>RSS</title><use xlink:href="/img/fa/all.svg#rss"/></svg></a></li></ul><div class=main-content><a href=https://creativecommons.org/licenses/by-nc-sa/4.0/>CC BY-NC-SA 4.0</a></div></footer></body></html>