<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta name=generator content="Hugo 0.123.3"><meta name=author content='Andrey Akinshin'><link href=/img/favicon.ico rel=icon type=image/x-icon><meta name=keywords content><title>Research projects | Andrey Akinshin</title>
<meta name=description content="Research projects Google Scholar ORCID ResearchGate arXiv Web of Science Scopus Statistical performance analysis Software benchmarking Mathematical models of gene networks Digital signal processing Statistical performance analysis This project aims to devel..."><meta name=twitter:site content="@andrey_akinshin"><meta name=twitter:creator content="@andrey_akinshin"><script>localStorage.getItem("color-theme")==="dark"||!("color-theme"in localStorage)&&window.matchMedia("(prefers-color-scheme: dark)").matches?document.documentElement.classList.add("dark"):document.documentElement.classList.remove("dark")</script><link href=https://aakinshin.net/css/main.min.25504b87a17cfaf1c000a5bcca42b1c4888fa4efc097d640d9139388b3e67e55.css rel=stylesheet type=text/css media=all><link rel=alternate type=application/rss+xml href=https://aakinshin.net/posts/index.xml title="RSS Feed"></head><body><nav><div class="container flex flex-wrap items-center justify-between mx-auto max-w-6xl h-12 px-2 xs:px-6"><div class="flex items-center justify-start justify-self-start"><div class="flex nav-item h-12 w-12 items-center justify-center"><a class=text-white href=https://aakinshin.net/><svg class="fai w-5 h-5 pr-0 mb-0.5"><use xlink:href="/img/fa/all.svg#house-chimney"/></svg></a></div><div class="flex nav-item h-12 items-center"><a class=text-white href=https://aakinshin.net/posts/>Posts</a></div><div class="flex nav-item h-12 items-center"><a class=text-white href=https://aakinshin.net/research/>Research</a></div><div class="flex nav-item h-12 items-center"><a class=text-white href=https://aakinshin.net/library/>Library</a></div><div class="flex nav-item h-12 items-center"><a class=text-white href=https://aakinshin.net/about/>About</a></div></div><button id=theme-toggle type=button title="Alt+Click to match OS color theme" class="nav-item p-3 content-center justify-self-end"><svg id="theme-toggle-dark-icon" class="hidden w-6 h-6" fill="currentcolor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001.0 1010.586 10.586z"/></svg><svg id="theme-toggle-light-icon" class="hidden w-6 h-6" fill="currentcolor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M10 2a1 1 0 011 1v1A1 1 0 119 4V3a1 1 0 011-1zm4 8a4 4 0 11-8 0 4 4 0 018 0zm-.464 4.95.707.707a1 1 0 001.414-1.414l-.707-.707a1 1 0 00-1.414 1.414zm2.12-10.607a1 1 0 010 1.414l-.706.707A1 1 0 1113.536 5.05l.707-.707a1 1 0 011.414.0zM17 11a1 1 0 100-2h-1a1 1 0 100 2h1zm-7 4a1 1 0 011 1v1a1 1 0 11-2 0v-1a1 1 0 011-1zM5.05 6.464A1 1 0 106.465 5.05l-.708-.707A1 1 0 004.343 5.757l.707.707zm1.414 8.486-.707.707a1 1 0 01-1.414-1.414l.707-.707a1 1 0 011.414 1.414zM4 11a1 1 0 100-2H3a1 1 0 000 2h1z" fill-rule="evenodd" clip-rule="evenodd"/></svg></button></div></nav><div class="main container mx-auto max-w-6xl px-6"><div class=main-content><h1 id=research-projects>Research projects</h1><div class="flex flex-wrap gap-y-1 mb-5"><a class=label-link href="https://scholar.google.com/citations?hl=en&user=rYVl83IAAAAJ&view_op=list_works&sortby=pubdate"><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#graduation-cap"/></svg>
Google Scholar
</a><a class=label-link href=https://orcid.org/0000-0003-3553-9367><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#orcid"/></svg>
ORCID
</a><a class=label-link href=https://www.researchgate.net/profile/Andrey_Akinshin><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#researchgate"/></svg>
ResearchGate
</a><a class=label-link href=https://arxiv.org/a/akinshin_a_1.html><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#arxiv"/></svg>
arXiv
</a><a class=label-link href=https://www.webofscience.com/wos/author/record/2102893><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#wos"/></svg>
Web of Science
</a><a class=label-link href="https://www.scopus.com/authid/detail.uri?authorId=56826126900"><svg class="fai fai-link"><use xlink:href="/img/fa/all.svg#scopus"/></svg>
Scopus</a></div><div class="flex flex-wrap gap-y-1 mb-5"><a class=label-link href=#statistics>Statistical performance analysis</a>
<a class=label-link href=#benchmarking>Software benchmarking</a>
<a class=label-link href=#gene-networks>Mathematical models of gene networks</a>
<a class=label-link href=#signal-processing>Digital signal processing</a></div><hr><h2 id=statistics>Statistical performance analysis</h2><p>This project aims to develop advanced statistical procedures
for the automatic analysis of software performance measurements.
This included various
location estimators,
dispersion estimators,
quantile estimators,
effect size estimators,
density estimators,
change point detectors,
outlier detectors,
and multimodality detectors.
The goal is to provide a set of non-parametric, robust, and efficient approaches
that are applicable to software performance distributions.</p><p>I regularly share my preliminary research notes in the form of
<a href=https://aakinshin.net/tags/statistics/>blog posts (195)</a>.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin</i> “Weighted quantile estimators” (2023) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/arXiv.2304.07265>10.48550/arXiv.2304.07265</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/arXiv.2304.07265 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2304.07265 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/2304.07265.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-wqe class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg> GitHub</a></div></li><li><i>Andrey Akinshin</i> “Finite-sample Rousseeuw-Croux scale estimators” (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/arXiv.2209.12268>10.48550/arXiv.2209.12268</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/arXiv.2209.12268 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2209.12268 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/2209.12268.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-frc class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg> GitHub</a></div></li><li><i>Andrey Akinshin</i> “Quantile absolute deviation” (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/ARXIV.2208.13459>10.48550/ARXIV.2208.13459</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/ARXIV.2208.13459 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2208.13459 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/2208.13459.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-qad class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg> GitHub</a></div></li><li><i>Andrey Akinshin</i> “Finite-sample bias-correction factors for the median absolute deviation based on the Harrell-Davis quantile estimator and its trimmed modification” (2022) // arXiv. DOI:&nbsp;<a href=https://doi.org/10.48550/ARXIV.2207.12005>10.48550/ARXIV.2207.12005</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Preprint</span>
<a href=https://dx.doi.org/10.48550/ARXIV.2207.12005 class="label-link text-sm">DOI</a>
<a href=https://arxiv.org/abs/2207.12005 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/2207.12005.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-mad-factors class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg> GitHub</a></div></li><li><i>Andrey Akinshin</i> “Trimmed Harrell-Davis quantile estimator based on the highest density interval of the given width” (2022) // Communications in Statistics - Simulation and Computation. Publisher: Taylor & Francis. Pp.&nbsp;1-11. DOI:&nbsp;<a href=https://doi.org/10.1080/03610918.2022.2050396>10.1080/03610918.2022.2050396</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1080/03610918.2022.2050396 class="label-link text-sm">DOI</a>
<a href=https://www.tandfonline.com/doi/abs/10.1080/03610918.2022.2050396 class="label-link text-sm">Link</a>
<a href=https://arxiv.org/abs/2111.11776 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/2111.11776.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href=https://github.com/AndreyAkinshin/paper-thdqe class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#github"/></svg> GitHub</a></div></li></ol><hr><h2 id=benchmarking>Software benchmarking</h2><p>This project aims to provide a set of reliable tools and approaches for software benchmarking.</p><p>Since 2013, I have been working on developing
<a href=https://github.com/dotnet/BenchmarkDotNet>BenchmarkDotNet</a>
(9K+ <a href=https://github.com/dotnet/BenchmarkDotNet/stargazers>GitHub stars</a>,
19K+ <a href="https://github.com/dotnet/BenchmarkDotNet/network/dependents?package_id=UGFja2FnZS0xNTY3MzExMzE%3D">dependent GitHub projects</a>,
24M+ <a href=https://www.nuget.org/packages/BenchmarkDotNet/>NuGet downloads</a>).
This .NET library helps to transform methods into benchmarks,
track their performance,
and share reproducible measurement experiments.
In the context of the <a href=/research/#statistics>statistical performance analysis project</a>,
I&rsquo;m working on the next-generation statistical engine for performance measurement analysis.</p><p>I also wrote a book <a href=https://aakinshin.net/prodotnetbenchmarking/>Pro .NET Benchmarking</a>
about good practices of performance measurements.
This book mostly covers
general benchmarking practices,
performance measurement analysis approaches,
and environmental factors that can affect obtained measurements.
It also contains dozens of case studies that demonstrate common benchmarking pitfalls and explain how to avoid them.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin</i> “Pro .NET Benchmarking” (2019) // Publisher: Apress. ISBN:&nbsp;978-1-4842-4940-6. Pp.&nbsp;662. DOI:&nbsp;<a href=https://doi.org/10.1007/978-1-4842-4941-3>10.1007/978-1-4842-4941-3</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Book</span>
<span class="label text-sm">Springer</span>
<a href=https://dx.doi.org/10.1007/978-1-4842-4941-3 class="label-link text-sm">DOI</a>
<a href=https://link.springer.com/book/10.1007/978-1-4842-4941-3 class="label-link text-sm">Springer</a>
<a href=https://www.apress.com/us/book/9781484249406 class="label-link text-sm">Apress</a></div></li><li><i>Andrey Akinshin</i> “The hard world of microbenchmarking” (2017) // Logeek Magazine. Vol.&nbsp;3. Pp.&nbsp;10–15.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<a href=https://career.luxoft.com/upload/iblock/74f/2283%5c_b2e%5c_logeek%5c_magazine%5c_3%5c_a4%5c_2017%5c_02%5c_27%5c_net.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li></ol><hr><h2 id=gene-networks>Mathematical models of gene networks</h2><p>This project aims to describe the mathematical model of various gene networks:
build systems of differential equations that simulate biological systems;
analyze corresponding dynamical systems;
find stationary points, limit cycles, and bifurcations;
develop software for numerical simulations.
The latest work is dedicated to the central regulatory circuit of the morphogenesis system of D. Melanogaster.</p><p>I was working on this project with fellows from
<a href=https://en.wikipedia.org/wiki/Sobolev_Institute_of_Mathematics>Sobolev Institute of Mathematics SB RAS</a> and
<a href=https://www.icgbio.ru/en/>Institute of Cytology and Genetics SB RAS</a>.
It was also the primary research topic of my PhD thesis
&ldquo;Mathematical and numerical modeling of gene network artificial regulatory circuits.&rdquo;</p><p><strong>Selected publications:</strong></p><ol reversed><li><i>V. P. Golubyatnikov, A. A. Akinshin, N. B. Ayupova, L. S. Minushkina</i> “Stratifications and foliations in phase portraits of gene network models” (2023) // Vavilov Journal of Genetics and Breeding. Publisher: Institute of Cytology and Genetics, SB RAS. Vol.&nbsp;26. No&nbsp;8. Pp.&nbsp;758–764. DOI:&nbsp;<a href=https://doi.org/10.18699/vjgb-22-91>10.18699/vjgb-22-91</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<a href=https://dx.doi.org/10.18699/vjgb-22-91 class="label-link text-sm">DOI</a></div></li><li><i>A. A. Akinshin, N. B. Ayupova, V. P. Golubyatnikov, N. E. Kirillova, O. A. Podkolodnaya, N. L. Podkolodnyy</i> “On a Numerical Model of a Circadian Oscillator” (2022) // Numerical Analysis and Applications. Vol.&nbsp;15. No&nbsp;3. Pp.&nbsp;187-196. DOI:&nbsp;<a href=https://doi.org/10.1134/S1995423922030016>10.1134/S1995423922030016</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1134/S1995423922030016 class="label-link text-sm">DOI</a>
<a href=https://rdcu.be/cUri9 class="label-link text-sm">Link</a></div></li><li><i>V. Golubyatnikov, A. Akinshin, N. Ayupova, N. Kirillova, N. Podkolodnyy, O. Podkolodnaya</i> “Mathematical and numerical modelling of the circadian oscillator” (2022) // Bioinformatics of Genome Regulation and Structure/Systems Biology (BGRS/SB-2022). Pp.&nbsp;1096–1096. DOI:&nbsp;<a href=https://doi.org/10.18699/SBB-2022-657>10.18699/SBB-2022-657</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.18699/SBB-2022-657 class="label-link text-sm">DOI</a></div></li><li><i>T.A. Bukharina, A.A. Akinshin, V.P. Golubyatnikov, D.P. Furman</i> “A Model of one central regulatory circuit” (2020) // The Twelfth International Multiconference 'Bioinformatics of genome regulation and structure/systems biology (BGRS/SB-2020)'. Publisher: Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of Sciences, Novosibirsk State University. Pp.&nbsp;143–144. DOI:&nbsp;<a href=https://doi.org/10.18699/BGRS/SB-2020-090>10.18699/BGRS/SB-2020-090</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.18699/BGRS/SB-2020-090 class="label-link text-sm">DOI</a>
<a href=https://publ.icgbio.ru/wp-content/uploads/2020/07/BGRS2020_15.07-144-145.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>T. A. Bukharina, A. A. Akinshin, V. P. Golubyatnikov, D. P. Furman</i> “Mathematical and Numerical Models of the Central Regulatory Circuit of the Morphogenesis System of Drosophila” (2020) // Journal of Applied and Industrial Mathematics. Vol.&nbsp;14. No&nbsp;2. Pp.&nbsp;249-255. DOI:&nbsp;<a href=https://doi.org/10.1134/S1990478920020040>10.1134/S1990478920020040</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1134/S1990478920020040 class="label-link text-sm">DOI</a>
<a href=https://rdcu.be/b5yyl class="label-link text-sm">Link</a>
<a href=https://link.springer.com/article/10.1134/S1990478920020040 class="label-link text-sm">Springer</a></div></li><li><i>A.A. Akinshin, V. P. Golubyatnikov, M.V. Kazantsev</i> “On discrete structures in phase portraits of some non-linear dynamical systems” (2016) // Siberian Electronic Mathematical Reports. Vol.&nbsp;13. Pp.&nbsp;A.23–A.24. DOI:&nbsp;<a href=https://doi.org/10.17377/semi.2016.13.020>10.17377/semi.2016.13.020</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=https://dx.doi.org/10.17377/semi.2016.13.020 class="label-link text-sm">DOI</a>
<a href=http://semr.math.nsc.ru/v13/a1-41.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>A.A. Akinshin, M.V. Kazantsev</i> “State transition diagrams in hypothetical models of cyclic gene networks functioning” (2015) // 5th International Young Scientists School “System Biology and Bioinformatics”. Novosibirsk. Pp.&nbsp;1.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/SBB2015/en class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> “A model of one biological 2-cells complex” (2014) // Geometry days in Novosibirsk - 2014: Abstracts of the International Conference. Novosibirsk. ISBN:&nbsp;978-5-86134-145-5. Pp.&nbsp;85–86.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> “An Extended Model of D. Melanogaster Machrochaete Morphogenesis Gene Network” (2014) // International Conference «Mathematical Modeling and High Performance Computing in Bioinformatics, Biomedicine and Biotechnology». Pp.&nbsp;10.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/mmhpc2014/ class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> “Modeling of Two-Cells Complex in Morphogenesis of D. Melanogaster Mechanoreceptors” (2014) // International Conference «Mathematical Modeling and High Performance Computing in Bioinformatics, Biomedicine and Biotechnology». Pp.&nbsp;11.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, V. A. Likhoshvai</i> “Symmetry in gene network models” (2014) // The 12th Bioinformatics Research and Education Workshop. Bielefeld, Germany. Center for Biotechnology — CeBiTec. Pp.&nbsp;1–5.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://wiki.techfak.uni-bielefeld.de/didy/brew class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin, T. A. Bukharina, D. P. Furman, V. P. Golubyatnikov</i> “Mathematical modeling of interaction of two cells in proneural cluster of D.mealnogaster (In Russian)” (2014) // Vestnik NSU [NSU Herald]. Novosibirsk. Vol.&nbsp;14. No&nbsp;4. Pp.&nbsp;3-10.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper%5c&amp;jrnid=vngu%5c&amp;paperid=350%5c&amp;option%5c_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> “Oscillating trajectories in some nonlinear dynamical systems” (2013) // International Conference «Differential Equations. Function Spaces. Approximation Theory». Novosibirsk. IM SB RAS. Pp.&nbsp;311.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> “Analysis of phase portraits in some gene networks models” (2013) // 5th International Young Scientists School “System Biology and Bioinformatics”. Pp.&nbsp;27.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> “Numerical analysis of gene networks models” (2013) // 8th FEBS Congress, Saint Petersburg, Russia, July 6-11, 2013. Vol.&nbsp;280. Pp.&nbsp;547. DOI:&nbsp;<a href=https://doi.org/10.1111/febs.12396>10.1111/febs.12396</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<a href=https://dx.doi.org/10.1111/febs.12396 class="label-link text-sm">DOI</a>
<a href="https://apps.webofknowledge.com/full%5c_record.do?product=UA%5c&amp;search%5c_mode=GeneralSearch%5c&amp;qid=1%5c&amp;SID=U1kYDgZoTuCyURIaqPx%5c&amp;page=1%5c&amp;doc=4" class="label-link text-sm">Web of Science</a>
<a href=https://publons.com/publon/34688524/ class="label-link text-sm">Publons</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, V. A. Likhoshvai</i> “Mathematical and computational models of gene networks functioning” (2013) // The 11th Bioinformatics Research and Education Workshop. Berlin, Germany. Computational Molecular Biology Department, Max Planck Institute for Molecular Genetics and IMPRS-CBSC.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://cmb.molgen.mpg.de/brew2013/ class="label-link text-sm">Conference site</a></div></li><li><i>A. A. Akinshin</i> “Andronov-Hopf bifurcation for some nonlinear delay equations (In Russian)” (2013) // Sibirskii Zhurnal Industrial'noi Matematiki [Siberian Journal of Industrial Mathematics]. Novosibirsk. Vol.&nbsp;XVI. No&nbsp;55. Pp.&nbsp;3–15.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://elibrary.ru/item.asp?id=20379316" class="label-link text-sm">RSCI</a>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper%5c&amp;jrnid=sjim%5c&amp;paperid=788%5c&amp;option%5c_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, I. V. Golubyatnikov</i> “On some multidimensional models of gene network functioning” (2013) // Journal of Applied and Industrial Mathematics. Publisher: Springer US. Vol.&nbsp;7. No&nbsp;3. Pp.&nbsp;296–301. DOI:&nbsp;<a href=https://doi.org/10.1134/S1990478913030022>10.1134/S1990478913030022</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm">Springer</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1134/S1990478913030022 class="label-link text-sm">DOI</a>
<a href="http://elibrary.ru/item.asp?id=20455452" class="label-link text-sm">RSCI</a>
<a href=http://link.springer.com/article/10.1134/S1990478913030022 class="label-link text-sm">Springer</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> “Non-uniqueness of cycles in gene networks models” (2012) // The eighth international conference on bioinformatics of genome regulation and structure / systems biology. ISBN:&nbsp;978-5-91291-011. Pp.&nbsp;28.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://www.bionet.nsc.ru/meeting/bgrs2012/BGRS2012%5c_Proceedings.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, Yu. A. Gaidov, I. V. Golubyatnikov</i> “Unstable cycles in gene networks models” (2012) // The eighth international conference on bioinformatics of genome regulation and structure / systems biology. ISBN:&nbsp;978-5-91291-011. Pp.&nbsp;29.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://www.bionet.nsc.ru/meeting/bgrs2012/BGRS2012%5c_Proceedings.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> “On Nonuniqueness of Cycles in Dissipative Dynamical Systems of Chemical Kinetics” (2012) // VI-th international conference Solitons, Collapses and Turbulence: Achievements, Developments and Perspectives.. Pp.&nbsp;71–72.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/files/conferences/sct2012/abstracts/114645/128115/Golubyatnikov.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>A. A. Akinshin</i> “Computer analysis of phase portraits in gene networks models” (2012) // Abstracts of Young scientist's school “Bioinformatics ans systems biology”. Pp.&nbsp;13.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span></div></li><li><i>A. A. Akinshin</i> “Research of discrete structures in some cyclic dynamical systems (In Russian)” (2012) // Polzunovskiy Vestnik [Polzunov Herald]. Barnaul. Vol.&nbsp;4. Pp.&nbsp;214–218.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href=http://elib.altstu.ru/elib/books/Files/pv2012%5c_04/pdf/214akinshin.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a>
<a href="http://elibrary.ru/item.asp?id=18947027" class="label-link text-sm">RSCI</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov</i> “Geometric characteristics of cycles in some symmetric dynamical systems (In Russian)” (2012) // Vestnik NSU [NSU Herald]. Series “Mathematics, mechanics, and informatics'”. Novosibirsk. Vol.&nbsp;12. No&nbsp;2. Pp.&nbsp;3–12.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">RU</span>
<span class="label text-sm">Article</span>
<a href="http://elibrary.ru/item.asp?id=18756520" class="label-link text-sm">RSCI</a>
<a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper%5c&amp;jrnid=vngu%5c&amp;paperid=114%5c&amp;option%5c_lang=eng" class="label-link text-sm">MathNet</a></div></li><li><i>A. A. Akinshin, V. P. Golubyatnikov, I. V. Golubyatnikov</i> “Mathematical and numerical modeling of gene network functioning” (2011) // International Conference “Modern Problems of Mathematics, Informatics and Bioinformatics”, devoted to the 100th anniversary of professor Alexei A. Lyapunov. Novosibirsk. ISBN:&nbsp;978-5-905569-03-6. Pp.&nbsp;81.<br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<a href=http://conf.nsc.ru/Lyap-100/ru/reportview/48237 class="label-link text-sm">Conference site</a>
<a href=http://conf.nsc.ru/files/conferences/Lyap-100/89299/Program-Thezises.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li></ol><hr><h2 id=signal-processing>Digital signal processing</h2><p>I was working on this project during my
postdoctoral research fellowship at the
<a href=https://www.weizmann.ac.il/>Weizmann Institute of Science</a>.</p><p><strong>Publications:</strong></p><ol reversed><li><i>Andrey Akinshin, Gil Goldman, Yosef Yomdin</i> “Geometry of error amplification in solving the Prony system with near-colliding nodes” (2021) // Mathematics of Computation. Vol.&nbsp;90. No&nbsp;327. Pp.&nbsp;267–302. DOI:&nbsp;<a href=https://doi.org/10.1090/mcom/3571>10.1090/mcom/3571</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Article</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1090/mcom/3571 class="label-link text-sm">DOI</a>
<a href=https://publons.com/publon/39312469/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1701.04058 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/1701.04058.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>Andrey Akinshin, Gil Goldman, Vladimir Golubyatnikov, Yosef Yomdin</i> “Accuracy of reconstruction of spike-trains with two near-colliding nodes” (2017) // 7th International Conference on Complex Analysis and Dynamical Systems (CA&DS VII). Vol.&nbsp;699. Pp.&nbsp;1–17. DOI:&nbsp;<a href=https://doi.org/10.1090/conm/699/14078>10.1090/conm/699/14078</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1090/conm/699/14078 class="label-link text-sm">DOI</a>
<a href=https://publons.com/publon/34688522/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1701.01482 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/1701.01482.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li><li><i>A. Akinshin, D. Batenkov, Y. Yomdin</i> “Accuracy of spike-train Fourier reconstruction for colliding nodes” (2015) // Sampling Theory and Applications (SampTA), 2015 International Conference on. Publisher: IEEE. Washington, DC. ISBN:&nbsp;978-1-4673-7353-1. No&nbsp;264. Pp.&nbsp;617–621. DOI:&nbsp;<a href=https://doi.org/10.1109/SAMPTA.2015.7148965>10.1109/SAMPTA.2015.7148965</a><br><div class="flex flex-wrap gap-y-1"><span class="label text-sm">EN</span>
<span class="label text-sm">Inproceedings</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#wos"/></svg> Web of Science</span>
<span class="label text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#scopus"/></svg> Scopus</span>
<a href=https://dx.doi.org/10.1109/SAMPTA.2015.7148965 class="label-link text-sm">DOI</a>
<a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=7148965" class="label-link text-sm">IEEE</a>
<a href=https://publons.com/publon/14766954/ class="label-link text-sm">Publons</a>
<a href=https://arxiv.org/abs/1502.06932 class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#arxiv"/></svg> arXiv</a>
<a href=https://arxiv.org/pdf/1502.06932.pdf class="label-link text-sm"><svg class="fai"><use xlink:href="/img/fa/all.svg#file-pdf"/></svg> PDF</a></div></li></ol></div></div><script>var themeToggleDarkIcon=document.getElementById("theme-toggle-dark-icon"),themeToggleBtn,themeToggleLightIcon=document.getElementById("theme-toggle-light-icon");const imagesDark=document.querySelectorAll(".img-dark"),imagesLight=document.querySelectorAll(".img-light");localStorage.getItem("color-theme")==="dark"||!("color-theme"in localStorage)&&window.matchMedia("(prefers-color-scheme: dark)").matches?(themeToggleLightIcon.classList.remove("hidden"),imagesDark.forEach(e=>{e.classList.remove("hidden")})):(themeToggleDarkIcon.classList.remove("hidden"),imagesLight.forEach(e=>{e.classList.remove("hidden")}));function toggleTheme(e){e?(document.documentElement.classList.add("dark"),themeToggleLightIcon.classList.remove("hidden"),themeToggleDarkIcon.classList.add("hidden"),imagesDark.forEach(e=>{e.classList.remove("hidden")}),imagesLight.forEach(e=>{e.classList.add("hidden")})):(document.documentElement.classList.remove("dark"),themeToggleDarkIcon.classList.remove("hidden"),themeToggleLightIcon.classList.add("hidden"),imagesLight.forEach(e=>{e.classList.remove("hidden")}),imagesDark.forEach(e=>{e.classList.add("hidden")}))}themeToggleBtn=document.getElementById("theme-toggle"),themeToggleBtn.addEventListener("click",function(){themeToggleDarkIcon.classList.toggle("hidden"),themeToggleLightIcon.classList.toggle("hidden"),imagesLight.forEach(e=>{e.classList.toggle("hidden")}),imagesDark.forEach(e=>{e.classList.toggle("hidden")}),event.altKey?(localStorage.removeItem("color-theme"),toggleTheme(window.matchMedia("(prefers-color-scheme: dark)").matches)):localStorage.getItem("color-theme")?localStorage.getItem("color-theme")==="light"?(document.documentElement.classList.add("dark"),localStorage.setItem("color-theme","dark")):(document.documentElement.classList.remove("dark"),localStorage.setItem("color-theme","light")):document.documentElement.classList.contains("dark")?(document.documentElement.classList.remove("dark"),localStorage.setItem("color-theme","light")):(document.documentElement.classList.add("dark"),localStorage.setItem("color-theme","dark"))}),window.matchMedia("(prefers-color-scheme: dark)").addEventListener("change",e=>{if(localStorage.getItem("color-theme")===null){const t=e.matches?"dark":"light";toggleTheme(t==="dark")}})</script><footer class="z-20 p-4 md:p-6 mt-3 w-full border-t border-gray-200 shadow flex items-center justify-center dark:border-gray-600 bg-white dark:bg-zinc-800"><span class="dark:text-gray-400 text-center">© 2013—2024 <span class=whitespace-nowrap>Andrey Akinshin</span></span><ul class="flex flex-wrap items-center px-4 mb-1"><li><a href=https://github.com/AndreyAkinshin><svg class="fai fai-link w-5 h-5"><title>GitHub</title><use xlink:href="/img/fa/all.svg#github"/></svg></a></li><li><a href=https://twitter.com/andrey_akinshin><svg class="fai fai-link w-5 h-5"><title>Twitter</title><use xlink:href="/img/fa/all.svg#twitter"/></svg></a></li><li><a href=https://aakinshin.net/posts/index.xml><svg class="fai fai-link w-5 h-5"><title>RSS</title><use xlink:href="/img/fa/all.svg#rss"/></svg></a></li></ul><div class=main-content><a href=https://creativecommons.org/licenses/by-nc-sa/4.0/>CC BY-NC-SA 4.0</a></div></footer></body></html>